pkg/mkverification-raw-efi/verify

```diff
@@ -416,6 +416,14 @@ if [ "$ARCH" = "x86_64" ]; then
     /vmcap > "$REPORT/vmcap.log"
 fi
 
+# verify the TPM
+if [ -c $TPM_DEVICE_PATH ]; then
+   logmsg "TPM device is present, running some extra tests"
+   /verifytpm.sh | tee -a "$REPORT/tpmchecks.log" >/dev/console 2>&1
```

Comment:
If there is a failure, do we indicate that in the summary.log?

Reply:
nope, I'll add it.